openlibrary/plugins/openlibrary/js/ile/index.js

```diff
@@ -41,6 +41,16 @@ export class IntegratedLibrarianEnvironment {
         // Ready bulk tagger:
         this.createBulkTagger()
 
+        this.$statusImages.on('click', 'img', (event) => {
+            const olid = $(event.currentTarget).attr('title');
+            const workLink = `/works/${olid}`;
+            window.open(workLink, '_blank');
+        });
+
+        this.$statusText.on('click', () => {
+            this.$statusImages.toggle();
+        });
+
```

Comment:
```suggestion
```
Nobody said anything about adding a click listener that opens a new page, and an `anchor` should be used to do that, anyway.